packages/@expo/cli/src/utils/validateApplicationId.ts

```diff
@@ -11,7 +11,7 @@ import { Log } from '../log';
 const debug = require('debug')('expo:utils:validateApplicationId') as typeof console.log;
 
 const IOS_BUNDLE_ID_REGEX = /^[a-zA-Z0-9-.]+$/;
-const ANDROID_PACKAGE_REGEX = /^[a-zA-Z][a-zA-Z0-9_]*(\.[a-zA-Z][a-zA-Z0-9_]*)+$/;
+const ANDROID_PACKAGE_REGEX = /^(?!.*\bnative\b)[a-zA-Z][a-zA-Z0-9_]*(\.[a-zA-Z][a-zA-Z0-9_]*)+$/;
```

Comment:
Was already accounted for here btw https://github.com/expo/expo/blob/5a90a55e5c9a110a94d9d899746af47b49bce619/packages/%40expo/cli/src/utils/validateApplicationId.ts#L79

Reply:
It wasn't working.